### PR TITLE
etcdserver: fix snapshot

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

File: integration/member_test.go [hunk 1]
```diff
@@ -15,9 +15,14 @@
...
 
 import (
+	"fmt"
 	"io/ioutil"
 	"os"
+	"reflect"
 	"testing"
+
+	"github.com/coreos/etcd/client"
+	"golang.org/x/net/context"
```
Comment (yichengq) [inline integration/member_test.go]: godep

Comment (xiang90) [inline integration/member_test.go]: fixed

**Comment by barakmich** [comment]
LGTM

File: integration/member_test.go [hunk 2]
```diff
@@ -74,3 +79,43 @@ func TestLaunchDuplicateMemberShouldFail(t *testing.T) {
...
+		kapi := client.NewKeysAPI(cc)
+		ctx, cancel := context.WithTimeout(context.Background(), requestTimeout)
+		key := fmt.Sprintf("foo%d", i)
+		resps[i], err = kapi.Create(ctx, "/"+key, "bar", -1)
+		if err != nil {
+			t.Fatalf("create on %s error: %v", m.URL(), err)
+		}
+		cancel()
+	}
+	m.Stop(t)
```
Comment (yichengq) [inline integration/member_test.go]: Could we check that it does have snapshot here?

**Pull request merged**